src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private Random randomColor = new Random();
+    private Color[] colors = Color.values();
+
+    public String getRandomColor() {
+        int randomIndex = randomColor.nextInt(colors.length);
+        return colors[randomIndex].name().toLowerCase();
```

Comment:
The requirement specifies to use the 'name()' method for getting the String representation of enum constants. Here you are using it correctly, but keep in mind that by calling 'toLowerCase()', you are changing the original representation. If the task requires the enum constant as it is, you should return 'colors[randomIndex].name();' without changing the case.